grin/src/Pipeline/Pipeline.hs

```diff
@@ -184,6 +187,7 @@ data PipelineStep
   | Eff EffectStep
   | T Transformation
   | Pass [PipelineStep]
+  | SimplePrintGrinH (Hidden (Doc -> Doc))
```

Comment:
Please unify the `SimplePrintGrinH` and `PrintGrinH` adding an extra parameter which tells how to render, Simple or not.

Reply:
Done.